app.js

```diff
@@ -1,101 +1,94 @@
-const env           = require('dotenv').config().parsed
-const parseString   = require('xml2js').parseString
-const mysql         = require('mysql')
-const eachOf        = require('async/eachOf')
-const NodeID3       = require('node-id3')
-const Twitter       = require('twitter')
-const rimraf        = require('rimraf')
-const {
-    createLogger,
-    format,
-    transports }    = require('winston')
-const {
-    combine,
-    timestamp,
-    printf }        = format
-
-
-var CronJob         = require('cron').CronJob
-var fs              = require('fs')
-var request         = require('request')
-var requestP        = require('request-promise-native')
-var TwCli           = new Twitter({
-                        consumer_key: process.env.TWITTER_CONSUMER_KEY,
-                        consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
-                        access_token_key: process.env.TWITTER_ACCESS_TOKEN_KEY,
-                        access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET
-                    });
+const env = require('dotenv').config().parsed
+
+const parseString = require('xml2js').parseString;
+const requestP    = require('request-promise-native');
+const CronJob     = require('cron').CronJob;
+const NodeID3     = require('node-id3');
+const concat      = require('concat-stream');
+const rimraf      = require('rimraf');
+const axios       = require('axios');
+const fs          = require('fs');
+
+const { 
+    log,
+    sanitizeContent,
+    sanitizeEpisode
+}                  = require('./lib/helpers');
+const TwController = require('./controllers/twitter.controller');
+const DbController = require('./controllers/db.controller');
+
+const TEST_CHANNEL = process.env.TEST_CHANNEL;
 
 const COVER = './assets/cover.jpg'
 const DDIR  = './downloads/'
-
-new CronJob('0 */30 * * * *', () => {
-    main()
-}, null, true)
+const CRON  = process.env.CRON;
+const ENV   = process.env.NODE_ENV;
+const DB    = new DbController();
```

Comment:
Nit: Use destructuring on `process.env`, it will be easier to add new env variables (and also to add default values).
```
const { CRON, ENV } = process.env;
```